programs/multisig/src/lib.rs

```diff
@@ -23,7 +23,7 @@ use anchor_lang::solana_program::instruction::Instruction;
 use std::convert::Into;
 
 #[program]
-pub mod serum_multisig {
+pub mod serum_multisig_impl {
```

Comment:
Happy to make this change. But can it be avoided by running `cargo test --lib` or something similar with `cargo test-bpf` ?

Reply:
It is possible to do `cargo test-bpf --manifest-path myprogram/Cargo.toml` to restrict it to just testing one crate, but that’s annoying to have to do every time. If you don’t like it here, I can remove it from the pull request, we can keep it in our fork instead.

Reply:
Reverting the change to this line here https://github.com/project-serum/multisig/pull/17 because it makes the build fail. This is an Anchor problem https://github.com/project-serum/anchor/issues/109.